LondonBritishLibrary/orient/BLorient501.xml

```diff
@@ -405,24 +443,32 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <term key="OldTestament"/>
                 </keywords>
             </textClass>
-            <langUsage><language ident="gez">Gǝʿǝz</language> <language ident="en">English</language></langUsage>
-            
+            <langUsage><language ident="gez">Gǝʿǝz</language> <language ident="en">English</language></langUsage>  
         </profileDesc>
-
-        
         <revisionDesc>
             <change who="ES" when="2017-02-01">Created catalogue entry</change>
             <change when="2020-11-26" who="DR">Added link to images</change>
             <change when="2022-06-01" who="DR">Added layout and collation</change>
             <change when="2022-06-29" who="DR">Added binding</change>
+            <change when="2023-10-23" who="CH">Added additions and extras, updated decoration, updated secondary bibliography, 
+                updated collation</change>
         </revisionDesc>
     </teiHeader>
     <facsimile>
         <graphic url="http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_501_f001r"/>
     </facsimile>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <ab/>
+           <div type="edition" corresp="a1" xml:lang="gez">
```

Comment:
The text of additions can be encoded in its entirety directly in the addition element, see other manuscripts for examples

Reply:
Ok. Done.